src/config.json

```diff
@@ -19,9 +19,9 @@
       "regions": [
         { 
           "id": "some-unique-id",
-          "left": 0, 
-          "top": 0, 
-          "zoomLevels": [0, 1, 2],
```

Comment:
for a particular region, don't we want to calculate multiple zoom levels? Is that being accomplished somewhere?

Reply:
Yes there is a for loop in `getTileIndices` - I though that that was the desired behavior, to provide one zoom level and get all the higher resolution ones "under" it.

Reply:
https://github.com/manzt/viv-tile-benchmark/pull/1/files#diff-6cf089419f06bbfa687fbcf29710f534R68-R83

Reply:
Ah sorry. Is that a part of the vendored code or a function you added? If added, maybe move outside of the module.